data/pathfinder/paizo/adventure_path/mummys_mask/mummys_mask_players_guide/mummys_mask_players_guide.pcc

```diff
@@ -1,34 +1,40 @@
 # CVS $Revision: 18863 $ $Author: amaitland $ -- Sat Dec  8 13:05:34 2007 -- reformated by prettylst.pl v1.38 (build 4387)
+# CVS $Revision: 18864 $ $Author: Lissa Azules $ -- Mon Jul 8 2019
```

Comment:
Repeat - white noise. We aren't using CVS or SVN, this should not be changed anymore.